drivers/SmartThings/zigbee-window-treatment/src/aqara/version/init.lua

```diff
@@ -0,0 +1,37 @@
+local clusters = require "st.zigbee.zcl.clusters"
+local aqara_utils = require "aqara/aqara_utils"
+
+local WindowCovering = clusters.WindowCovering
+local AnalogOutput = clusters.AnalogOutput
+
+local APPLICATION_VERSION = "application_version"
+
+local function shade_level_report_legacy_handler(driver, device, value, zb_rx)
+  aqara_utils.emit_shade_level_event(device, value)
+  aqara_utils.emit_shade_event(device, value)
+end
+
+local function shade_level_report_handler(driver, device, value, zb_rx)
+  aqara_utils.emit_shade_level_event(device, value)
+  aqara_utils.emit_shade_event(device, value)
+end
+
+local aqara_window_treatment_version_handler = {
+  NAME = "Aqara Window Treatment Version Handler",
+  zigbee_handlers = {
+    attr = {
+      [WindowCovering.ID] = {
+        [WindowCovering.attributes.CurrentPositionLiftPercentage.ID] = shade_level_report_legacy_handler
```

Comment:
based on how this is written, it might make sense to make the base driver handle this attribute and then invert the `can_handle` of this driver to be `version ~= 34` and only in that case ignore the value